.github/dependabot.yml

```diff
@@ -0,0 +1,22 @@
+# https://docs.github.com/en/code-security/supply-chain-security/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    labels: [ ]     # Do not add default "dependency" label
```

Comment:
Can we add the `tooling` label?

Reply:
Yes we can, but updating dependencies for security reasons isn't tooling. More of an `improvement`, but in general, we don't put labels on standalone PRs

Reply:
Ah, this is in github action dependencies (not visible in UI 🙂 ), so yes, we could, be we're not putting labels on standalone PRs so we can be without it